src/router/routes/process-create.tsx

```diff
@@ -1,29 +1,26 @@
 import { lazy } from 'react'
 // These aren't lazy loaded since they are main layouts and related components
 import LayoutProcessCreate from '~elements/LayoutProcessCreate'
+import ProtectedRoutes from '~src/router/ProtectedRoutes'
 import { Routes } from '.'
 import { SuspenseLoader } from '../SuspenseLoader'
 
-const ProtectedRoutes = lazy(() => import('../ProtectedRoutes'))
 const ProcessCreate = lazy(() => import('~elements/dashboard/processes/create'))
 
 const ProcessCreateElements = [
   {
-    element: (
-      <SuspenseLoader>
-        <ProtectedRoutes />
-      </SuspenseLoader>
-    ),
-    children: [
-      {
-        path: Routes.processes.create,
-        element: (
-          <SuspenseLoader>
-            <ProcessCreate />
-          </SuspenseLoader>
-        ),
-      },
-    ],
+    ...ProtectedRoutes({
+      children: [
+        {
+          path: Routes.processes.create,
+          element: (
+            <SuspenseLoader>
+              <ProcessCreate />
+            </SuspenseLoader>
+          ),
+        },
+      ],
+    }),
```

Comment:
Changing the method as I stated before, this could be simply:

```suggestion
    ...ProtectedRoutes([
      {
        path: Routes.processes.create,
        element: (
          <SuspenseLoader>
            <ProcessCreate />
          </SuspenseLoader>
        ),
      },
    ]),
```

Reply:
b18a0c9